packages/runner/bin/runner.js

```diff
@@ -48,20 +52,13 @@ import setupServer from '../src/setupServer.js';
   console.log(chalk.gray('Starting runner with your game...\n'));
   console.log('running with options:', options);
 
-  const cleanupServerFunc = await setupServer({
-    isEmptyBackend: options.emptyBackend,
-    apiPort: portForRunnerBackend,
-  });
+  const cleanupServerFunc = await setupServer({ apiPort: portForRunnerBackend });
 
   let cleanupFrontendsFunc;
   let runnerFrontendProcess;
-  let userFrontendProcess;
 
   if (options.dev) {
-    userFrontendProcess = exec(`cd test_app/frontend && BROWSER=None PORT=${portForUserFrontend} npm start`);
-    userFrontendProcess.stdout.pipe(process.stdout);
-
-    runnerFrontendProcess = exec(`cd frontend && PORT=${portForRunnerFrontend} REACT_APP_USER_PORT=${portForUserFrontend} REACT_APP_BACKEND_PORT=${portForRunnerBackend} npm start`);
```

Comment:
`cd frontend` is a relative path, so when running the `runner` in another package, this `frontend` folder will NOT be `runner/frontend`. Because of this, I changed it to the absolute path using the current script dir. See how `runnerFrontendPath` is set